### PR TITLE
test(acceptance): adding python 3.8 acceptance tests

### PR DESCRIPTION
**Pull request by galbash**

### CODE REVIEW

File: acceptance/run.sh [hunk 1]
```diff
@@ -12,9 +13,7 @@ function run_acceptance_test() {
     serverless remove --runtime ${runtime} --runtimeName ${runtimeName} --buildNumber ${build_num}
 }
 
-run_acceptance_test python2.7 p27
-run_acceptance_test python3.6 p36
-#run_acceptance_test python3.7 p37
+run_acceptance_test python${version} py${version//.}
```
Comment (henprtz) [inline acceptance/run.sh]: py? not p ?

File: .travis.yml [hunk 2]
```diff
@@ -11,14 +14,23 @@ install:
 
 script:
   - "pylint --msg-template='{path}:{line}: [{msg_id}({symbol}) {obj}] {msg}' epsagon/"
-  - tox
+  - pytest
```
Comment (henprtz) [inline .travis.yml]: explain?

File: .travis.yml [hunk 3]
```diff
@@ -11,14 +14,23 @@ install:
...
   - npm install && export PATH=$(pwd)/node_modules/.bin:$PATH
-  - ./acceptance/run.sh $TRAVIS_BUILD_NUMBER
+  - ./acceptance/run.sh $TRAVIS_BUILD_NUMBER $TRAVIS_PYTHON_VERSION
 
+jobs:
+  include:
+    - stage: build-and-deploy
+      provider: script
+      python:
+        - "3.8"
```
Comment (ronyis) [inline .travis.yml]: What's that used for?
It doesn't override the global Python version?

Comment (galbash) [reply .travis.yml]: yes, only for deploy

File: .travis.yml (hunk 2 above)
Comment (galbash) [reply .travis.yml]: no need, we now have a separate machine for each build

File: acceptance/run.sh (hunk 1 above)
Comment (galbash) [reply acceptance/run.sh]: works idk

**Pull request merged**